packages/blocks/azure/src/lib/actions/custom-azure-api-action.ts

```diff
@@ -0,0 +1,17 @@
+import { createCustomApiCallAction } from '@openops/blocks-common';
+import { authenticateUserWithAzure, azureAuth } from '@openops/common';
+
+export const customAzureApiCallAction = createCustomApiCallAction({
+  auth: azureAuth,
+  name: 'custom_azure_api_call',
+  description: 'Make a custom REST API call to Azure',
+  displayName: 'Custom Azure API Call',
+  baseUrl: () => 'https://management.azure.com?api-version=2024-10-21',
```

Comment:
it doesnt work without version, i think it makes sense to add it in the baseurl, alternatively we can add it to the description

Reply:
we dont normally add the / in other custom apis, should i still do it?